components/Reviews/EditReviewContent/NewEditTemplate/index.tsx

```diff
@@ -17,11 +18,13 @@ interface FormValues {
 interface NewReviewTemplateProps {
   review: ReviewData;
   updateReviewCard: (updatedReview: ReviewData) => void;
+  id: any;
 }
 
-const NewReviewTemplate: React.FC<NewReviewTemplateProps> = ({
+const NewReviewTemplate: React.FC<NewReviewTemplateProps & { id: string }> = ({
```

Comment:
```suggestion
const EditReviewTemplate: React.FC<EditReviewTemplateProps & { id: string }> = ({
```

also if you want to add the id as prop just add it to EditReviewTemplateProps

Reply:
I'm confused I already had it this way